articles/cosmos-db/mongodb-change-streams.md

```diff
@@ -49,6 +49,9 @@ var cursor = db.coll.watch(
 
 ```
 
+> __NOTE__ Ensure your Cosmos DB instance is provisioned with at least version 3.6 of Cosmos DB’s API for MongoDB. If you run the example against an earlier version, you may see the error `Unrecognized pipeline stage name: $changeStream`
```

Comment:
@khowling Thanks for the suggestion. can you please move the note to the top of the page. It fits best after the first paragaph. Also suggesting some more updates to the note to fit the content style. 

> [!NOTE]
> To use change streams, make sure to create the account with at least version 3.6 of Azure Cosmos DB’s API for MongoDB. If you run the chnage stream examples against an earlier version, you may see the `Unrecognized pipeline stage name: $changeStream` error.